project/Base.scala

```diff
@@ -32,7 +32,7 @@ object Base {
 class Base extends Build {
   import Base._
 
-  val headVersion = "1.1.2-medallia-1.8.1"
+  val headVersion = "1.1.2-medallia-1.8.1-customResponseClassifier"
```

Comment:
Shouldn't this be `1.8.2` or similar?

Reply:
Well, this is actually a new feature, so it should be a 1.9.0, but that's already taken. If you prefer updating the patch instead of a custom modifier, let me know.

Reply:
No, I'm fine with `1.9.0`, it's just that I'm not sure how that gets stamped here.